### PR TITLE
BZ-1782034: Clarifying to use the same etcd snapshot upon restore

### PR DESCRIPTION
**Pull request by bergerhoffer**

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@openshift/team-documentation For peer review please!

**Comment by bergerhoffer** [comment]
@retroflexer Can you take a look please?

File: modules/backup-etcd.adoc [hunk 1]
```diff
@@ -7,6 +7,8 @@
 
 Follow these steps to back up etcd data by creating a snapshot. This snapshot can be saved and used at a later time if you need to restore etcd.
 
+You should only save a snapshot from a single master host, not one from each master host in the cluster.
```
Comment (mburke5678) [inline modules/backup-etcd.adoc]: Is _one_ a logic error that the translation team talked about yesterday? 

_You should only save a snapshot from a single master host. You do not need a snapshot from each master host in the cluster._ ??

**Comment by mburke5678** [comment]
@bergerhoffer One nit. Otherwise LGTM

File: modules/backup-etcd.adoc (hunk 1 above)
Comment (bergerhoffer) [reply modules/backup-etcd.adoc]: Thanks for the suggestion @mburke5678! I wasn't able to watch that presentation but was looking forward to watching the recording.

**Comment by bergerhoffer** [comment]
@geliu2016 Can you please review?

**Comment by geliu2016** [comment]
LGTM, thanks

**Pull request merged**